### DISCUSSION TITLE
How do I build and run a WebView2 sample from the command line, without using the Visual Studio GUI?

### DISCUSSION THREAD

**Discussion by yg-i** [MicrosoftEdge/WebView2Feedback#4595] [Q&A]
The guide here is absolutely insistent that I have to use Visual Studio:

https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/win32

But I'd really, really rather not use Visual Studio. I do have Visual Studio installed on my machine. But only for the build tools,. I much prefer using the command line cl.exe in a developer powershell.

Is there any simple sequence of purely command-line based steps I can take to get, say, the Win32_GettingStarted example to build?

Thanks in advance!

**Comment by Blouet** [comment]
I wish someone would answer this as I am in the same boat. Microsoft have a great tool here but to use it appears that VS is the only route. What a shame!!!
